src/gl/vbo_mesh.js

```diff
@@ -12,6 +12,7 @@ export default class VBOMesh  {
         this.gl = gl;
         this.vertex_data = vertex_data; // typed array
         this.vertex_layout = vertex_layout;
+        this.vertex_elements = options.vertex_elements;
         this.buffer = this.gl.createBuffer();
```

Comment:
I think `vertex_buffer` is a little confusing, as this suggest a buffer of vertices, which is what's stored in `vertex_data`. We could rename `vertex_data` to `vertex_buffer` and `vertex_elements` to `element_buffer` or `indices_buffer`.